src/components/molecules/TagStyleFilter.js

```diff
@@ -210,6 +211,7 @@ const TagStyledFilter = props => {
   const resetFilters = () => {
     applyFilter(initialFiltersState)
   }
+  useEffect(() => applyFilter(filters), [onListFiltered])
```

Comment:
eslint errors here:
 `React Hook useEffect has missing dependencies: 'applyFilter' and 'filters'. Either include them or remove the dependency array`